### ISSUE TITLE
There seems to be a problem with P2P with multiple outputs

### ISSUE THREAD

**Issue by Desempregago** [beecupbe/ae2uel_extended#20]
**Describe the bug**
The input P2P seems to "stop sending" information/items/energy/fluids whenever you connect an additional output to the network.

**To Reproduce**
Place 3 P2Ps, set one as input and one as output: everything works, but as soon as you connect the second output, the input simply "stops". All of the three P2Ps are online and have available channels to them.

**Expected behavior**
The second and subsequent output P2P nodes to work as the first one did.

**Environment**
- Minecraft Version: 1.12.2
- AE2 Version: rv6-stable-7
- Forge Version: 14.23.5.2860

**Comment by Desempregago** [comment]
Nevermind, it was something wrong with my power generation.

**Comment by Desempregago** [comment]
Yeah, looks like the power generation wasn't the problem...[Video Proof](https://youtu.be/C4YEqkWmYCc)

**Comment by beecupbe** [comment]
try testing it in the original version of AE2 Unofficial Extended Life.

**Comment by Desempregago** [comment]
Just tested, happens with the original version as well.

**Comment by beecupbe** [comment]
it looks like the maximum number of outputs is 1.
![pic](https://github.com/beecupbe/ae2uel_extended/assets/115642276/fd4086e2-f255-47c8-8c43-53a5fcc442a1)

**Comment by Desempregago** [comment]
isn't it only the WAILA compat?
